.golangci.yml

```diff
@@ -0,0 +1,33 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  gocyclo:
+    min-complexity: 10
+  maligned:
+    suggest-new: true
+  goconst:
+    min-len: 3
+    min-occurrences: 3
+  misspell:
+    locale: US
+  lll:
+    line-length: 120
+
+linters:
+  enable-all: true
+  disable:
```

Comment:
Did you disable these based on the complexity of resolving errors with all enabled by default ? I can imagine that the example file might be causing some severe issues.